pytest.ini

```diff
@@ -8,6 +8,7 @@ markers =
     bucket_policy
     bucket_encryption
     bucket_logging
+    bucket_logging_cleanup
```

Comment:
this one is unused?

Reply:
what do you mean by "unused"? it is in the `s3tests_boto3/functional/test_s3.py`
currently I'm not using it as a filter in any of the teuthology code.